### DISCUSSION TITLE
Question about RegressionKrigging

### DISCUSSION THREAD

**Discussion by sachin9390** [GeoStat-Framework/PyKrige#239] [Q&A] [answered]
can we write the output from RegressionKrigging  on grid/raster and could you please explain these  **args, **kwargs ?

**Comment by MuellerSeb** [comment, marked as answer]
Hi there!

Take a look at [write_asc_grid](https://geostat-framework.readthedocs.io/projects/pykrige/en/stable/generated/pykrige.kriging_tools.write_asc_grid.html#pykrige.kriging_tools.write_asc_grid) to export raster files.

What specific `*args` and `**kwargs` are talking about? Maybe [this](https://book.pythontips.com/en/latest/args_and_kwargs.html) already helps.

Sebastian
